### PR TITLE
added mocha at the test environment level. Also added mocha test script

### PR DESCRIPTION
**Pull request by MrJnrman**

### CODE REVIEW

File: tests/sample.test.js [hunk 1]
```diff
@@ -14,19 +14,35 @@ export default {
     browser.maximizeWindow();
   },
 
-  'Testing the Title of QualityWorks Homepage': browser => {
+  'Testing the Title of Different Quality Works Pages': browser => {
```
Comment (xootwa) [inline tests/sample.test.js]: hey please use the following format for your tests

'Test Name'(browser){
 ....
}

Reason: This isn't the best use of arrow functions

Comment (romarioraffington) [inline tests/sample.test.js]: * "QualityWorks" should be one word
* Make description more specific. What page are you testing?

File: page_objects/blog.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    // XPATH Selector
+    firstResultFromSearch: { selector: ".//*[@id='post-211804']/div/div[2]/header/h2/a", locateStrategy: 'xpath'}
+}
+
+const commands = {
+
+    searchQualityWorksBlog() {
+        return this
+        .waitForElementVisible('@searchBox', 'Searchbox is visible')
```
Comment (xootwa) [inline page_objects/blog.js]: use waitThenSetValue/waitedSetValue to eliminate unneeded lines

.waitForElementVisible('@searchBox', 'Searchbox is visible')
..

File: tests/sample.mocha.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+describe('Mocha using nightwatch', function () {
```
Comment (xootwa) [inline tests/sample.mocha.test.js]: Give the Describe a better title like: 'Menu Bar'
Give the it a better title like: 'Should have correct url'

**Pull request merged**